lib/ecto/changeset.ex

```diff
@@ -823,7 +833,7 @@ defmodule Ecto.Changeset do
     end
   end
 
-  defp on_cast_default(type, module) do
+  def on_cast_default(type, module) do
```

Comment:
Move this function to Ecto.Changeset.Relation so we can share it from both modules. :)